package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@personal-api/plugin-instagram",
-  "version": "0.9.0",
-  "description": "Fetch recent Instagram photos for a user.",
+  "version": "0.9.1",
+  "description": "Add an Express route that returns a user's Instagram photos.",
   "main": "index.js",
   "scripts": {
     "test": "npm run lint && npm run unit",
     "lint": "xo",
     "unit": "nyc ava"
   },
   "author": "Chris Vogt <mail@chrisvogt.me>",
+  "repository": "github:personal-api/plugin-instagram",
```

Comment:
This was added to silence a _package.json_ validation warning logged during `npm install`.